Sources/Internal/FrameProvider.swift

```diff
@@ -140,13 +140,15 @@ final class FrameProvider {
     let date = calendar.startDate(of: day)
     let dayOfWeekPosition = calendar.dayOfWeekPosition(for: date)
     let rowInMonth = adjustedRowInMonth(for: day)
+    let numberOfWeekRows = rowInMonth + 1
```

Comment:
```suggestion
    let numberOfWeekRowsBeforeDay = rowInMonth + 1
```
Just a thought to improve clarity. At first when reading this I thought it was total number of week rows in the month.

Reply:
Changed the name to `numberOfWeekRowsThroughDay` since it's inclusive (not up-to) the the row the day is in